faiss/CMakeLists.txt

```diff
@@ -149,7 +149,12 @@ if(NOT WIN32)
 endif()
 
 if(FAISS_OPT_LEVEL STREQUAL "avx2")
-  target_compile_options(faiss PRIVATE $<$<COMPILE_LANGUAGE:CXX>:-mavx2 -mfma -mf16c -mpopcnt>)
+  if(NOT WIN32)
+    target_compile_options(faiss PRIVATE $<$<COMPILE_LANGUAGE:CXX>:-mavx2 -mfma -mf16c -mpopcnt>)
+  else()
```

Comment:
Nit: Maybe reverse the order (`if(WIN32) ... else()`).

Reply:
I can do it, but I think it's actually more consistent like that:
* most of the CMake-files deal with win/non-win like this (see e.g. a few lines above)
* the unix flags should come first, because they determine the goal, and the windows part argues why it doesn't have all the those flags (so reversing the order would make it harder to read logically, IMO).